test/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter_test.cc

```diff
@@ -47,16 +47,27 @@ class GrpcJsonTranscoderConfigTest : public testing::Test {
 public:
   const envoy::config::filter::http::transcoder::v2::GrpcJsonTranscoder
   getProtoConfig(const std::string& descriptor_path, const std::string& service_name,
+                 const std::vector<std::string>& ignored_query_parameters,
                  bool match_incoming_request_route = false) {
     std::string json_string = "{\"proto_descriptor\": \"" + descriptor_path +
                               "\",\"services\": [\"" + service_name + "\"]}";
     auto json_config = Json::Factory::loadFromString(json_string);
     envoy::config::filter::http::transcoder::v2::GrpcJsonTranscoder proto_config;
     Envoy::Config::FilterJson::translateGrpcJsonTranscoder(*json_config, proto_config);
     proto_config.set_match_incoming_request_route(match_incoming_request_route);
+    for (const std::string& query_param : ignored_query_parameters) {
+      proto_config.add_ignored_query_parameters(query_param);
+    }
+
     return proto_config;
   }
 
+  const envoy::config::filter::http::transcoder::v2::GrpcJsonTranscoder
```

Comment:
I think you can live without this definition by `const std::vector<std::string>& ignored_query_parameters = {}`? or just add {} to every callee site.

Reply:
Done.